packages/cli/src/lifecycles/serve.js

```diff
@@ -1,61 +1,109 @@
-/* eslint-disable complexity */
-// TODO ^^^
-const { promises: fsp } = require('fs');
+const fs = require('fs');
 const path = require('path');
 const Koa = require('koa');
 
-const Transform = require('../transforms/transform.interface');
-const HTMLTransform = require('../transforms/transform.html');
-const MarkdownTransform = require('../transforms/transform.md');
-const CSSTransform = require('../transforms/transform.css');
-const JSTransform = require('../transforms/transform.js');
-const JSONTransform = require('../transforms/transform.json.js');
-const AssetTransform = require('../transforms/transform.assets');
+const pluginNodeModules = require('../plugins/resource/plugin-node-modules');
+const pluginResourceStandardCss = require('../plugins/resource/plugin-standard-css');
+const pluginResourceStandardFont = require('../plugins/resource/plugin-standard-font');
+const pluginResourceStandardHtml = require('../plugins/resource/plugin-standard-html');
+const pluginResourceStandardImage = require('../plugins/resource/plugin-standard-image');
+const pluginResourceStandardJavaScript = require('../plugins/resource/plugin-standard-javascript');
+const pluginResourceStandardJson = require('../plugins/resource/plugin-standard-json');
+const { ResourceInterface } = require('../lib/resource-interface');
+const pluginUserWorkspace = require('../plugins/resource/plugin-user-workspace');
 
 function getDevServer(compilation) {
   const app = new Koa();
+  const compilationCopy = Object.assign({}, compilation);
+  const resources = [
+    // Greenwood default standard resource plugins
+    pluginResourceStandardCss.provider(compilationCopy),
+    pluginResourceStandardFont.provider(compilationCopy),
+    pluginResourceStandardHtml.provider(compilationCopy),
+    pluginResourceStandardImage.provider(compilationCopy),
+    pluginResourceStandardJavaScript.provider(compilationCopy),
+    pluginResourceStandardJson.provider(compilationCopy),
+
+    // custom user resource plugins
+    ...compilation.config.plugins.filter((plugin) => {
```

Comment:
My only concern here is we're no longer accepting pre-transform(resource) plugins and we're defaulting them all to pos transforms.  I don't have an immediate pre-default use case but for now I think this is okay.  Also another issue might be that nobody can override the default transforms(resources).